### PR TITLE
{chart,scripts}: move datasource test to helm chart

### LINKED ISSUE #517: Re-enable testing datasources
**What is missing?**

We have disabled testing for datasources correctness as per https://github.com/timescale/tobs/blob/main/.github/workflows/tests.yml#L55-L58.

**Why do we need it?**

To ensure datasources are properly working.

**Anything else we need to know?**:

We can convert this script and make target into a helm test - https://helm.sh/docs/topics/chart_tests/. Example of such approach - https://github.com/parca-dev/helm-charts/blob/master/charts/parca/templates/tests/test-connection.yaml

### LINKED ISSUE #519: Datasources connections are broken.
**What happened?**

Installed tobs from `main` branch and port-forwarded to grafana to check a dashboard. Dashboards had connectivity issues due to misconfigured promscale (promql) datasource.

Grafana errors out with:
> Templating
Template variable service failed Get "http://test-promscale-connector.default.svc:9201/api/v1/series?match%5B%5D=up%7Bjob%3D%22kubelet%22%2C%20metrics_path%3D%22%2Fmetrics%2Fcadvisor%22%7D&start=1659346071&end=1659349671": dial tcp: lookup test-promscale-connector.default.svc on 10.96.0.10:53: no such host

**Did you expect to see something different?**

I expected dashboard to work.

### PR DESCRIPTION
**Pull request by paulfantom**
## Description

Alternative approach to #520 which allows to use `helm test` command to test datasource connectivity. This method may also be used later on to test data ingestion (issue https://github.com/timescale/tobs/issues/324).

Re-enable datasource test, fixes #517 
Fix promql datasource in grafana, fixes #519 

## Type of change

*What type of changes does your code introduce to tobs? Put an `x` in the box that apply.*

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [x] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

### CODE REVIEW

**Review by onprem — COMMENTED** [review]
As `helm test` is run as part of `ct install`, I guess we can remove the GH action and just rely on the test to be present to verify datasources, which is nice.

Since the check-datasources.sh script got removed, let's cleanup the Makefile target as well.

**Comment by paulfantom** [comment]
This PR is now doing a bit more than I initially anticipated. In addition to what was primarily being fixed it also changes the following:

- CI helm values are limited only to necessary overrides
- additional `lint` and `e2e` make targets were added and now they are also used in CI

File: chart/ci/default-values.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This is the default values for the chart to run in CI system and it can be used as
+# a base template for other *-values.yaml files. Only resource requests are nullified
+# to allow starting the stack on github action runner and prevent issues related
+# to pod scheduling due to insufficient host resources.
+
+timescaledb-single:
```
Comment (onprem) [inline chart/ci/default-values.yaml:6]: I think we also need the environment variables to disable telemetry here.

**Review by onprem — COMMENTED** [review]
LGTM, let's just address the missing env variables.

File: chart/ci/default-values.yaml (hunk 1 above)
Comment (paulfantom) [reply chart/ci/default-values.yaml:6]: Good catch! I added now.

**Review by onprem — APPROVED** [review]

**Pull request merged**